contributing/documentation/overview.rst

```diff
@@ -136,9 +136,38 @@ even remove any content and do your best to comply with the
 
 **Step 6.** **Push** the changes to your forked repository:
 
+Rebase your branch
```

Comment:
adding a heading here breaks the steps.

the case of rebasing should be a section later in the doc, not in the quick start guide for your first contribution (and maybe it could link to the documentation about rebasing a PR for code contribution instead of duplicating it, as it works exactly the same, given that this is purely about git usage)

Reply:
I agree with the idea of just using a link.
However, I believe it would be useful here so that beginners don't feel lost if they face this case. Be it because they began working days or weeks ago, or just minutes ago and by chance a PR got merged just in the meantime.

Reply:
@stof you are right. We moved this new content below and integrated in one of the existing steps to make it more natural. Thanks.